src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type FooterProps = {
+  counter: number;
+  filter: Filter;
+  setFilter: (filter: Filter) => void;
```

Comment:
you can copy type of setState function, when you hover over that function in VSC, it will be more specific than void